src/main/java/carpet/script/Fluff.java

```diff
@@ -125,15 +125,7 @@ public boolean numParamsVaries() {
 
         public List<Value> unpackArgs(List<LazyValue> lzargs, Context c, Context.Type contextType)
         {
-            List<Value> args = new ArrayList<>();
-            for (LazyValue lv : lzargs)
-            {
-                Value arg = lv.evalValue(c, contextType);
-                if (arg instanceof FunctionUnpackedArgumentsValue)
-                    args.addAll(((ListValue) arg).getItems());
-                else
-                    args.add(arg);
-            }
+            List<Value> args = unpackLazy(lzargs, c, contextType);
```

Comment:
sole reason for me to inline it was to save on an extra function call in that crucial spot that is passed for every system call. Probably with zero impact and I shouldn't be so particular about it. Thoughts?

Reply:
If I've learnt something from being in CaffeineMC's discord for so long, is that if it gets called frequently enough the JVM (or JIT or whatever) will inline it (would have to check it's under the bytecode limit, but a static method invocation doesn't seem that bad anyway IMO).

Reply:
ok then. Didn't know how deep jit gets. Thanks for clarification.